front/app/api/project_folder_moderators/keys.ts

```diff
@@ -0,0 +1,17 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = { type: 'user', variant: 'folder_moderator' };
+
+const projectFolderModeratorsKeys = {
+  all: () => [baseKey],
+  lists: () => [{ ...baseKey, operation: 'list' }],
+  list: ({ projectFolderId }: { projectFolderId: string }) => [
```

Comment:
In some other files, I notice we use "QueryParameters" for the list params - is that only used if we have more than one parameter??

` list: (parameters: IQueryParameters)`

Reply:
Yes, I think it's a bit of a judgement call but for only one string, it doesn't feel worth it to create an interface and export / import it.. But to be clear, there's also a preference element to this so overall, I don't think we have a super strict rule here.